actionpack/test/dispatch/system_testing/driver_test.rb

```diff
@@ -161,6 +161,21 @@ class DriverTest < ActiveSupport::TestCase
     ::Selenium::WebDriver::Chrome::Service.driver_path = original_driver_path
   end
 
+  test "preloads browser's driver_path with DriverFinder if a path isn't already specified" do
+    original_driver_path = ::Selenium::WebDriver::Chrome::Service.driver_path
+    ::Selenium::WebDriver::Chrome::Service.driver_path = nil
+
+    # Our stub must return a path to a real executable, otherwise an internal Selenium assertion will fail.
+    found_executable = RbConfig.ruby
+    ::Selenium::WebDriver::DriverFinder.stub(:path, found_executable) do
+      ActionDispatch::SystemTesting::Driver.new(:selenium, screen_size: [1400, 1400], using: :chrome)
+    end
```

Comment:
I'm hesitant to stub `DriverFinder.path` since we're relying on it directly.  Does this test blow up if we don't stub?  Perhaps it would be slightly better to stub `SeleniumManager.driver_path`, which `DriverFinder.path` calls?

Reply:
I used a stub to avoid a network call to download chromedriver, which would slow down the test and perhaps make it less reliable. However I agree that the test is more realistic and valuable without the stub. I'll remove it.

Reply:
Addressed in be39f0c695

Reply:
> to avoid a network call to download chromedriver

Ah, right, in that case we _should_ stub.  (Sorry! :pray:)  But I think stubbing `SeleniumManager.driver_path` might be a better choice because if the behavior of `DriverFinder.path` changes, then we will be more likely to catch it.

Reply:
OK, addressed in 3f089b3aa1 ✅